core/coredns.go

```diff
@@ -1,4 +1,7 @@
 // Package core registers the server and all plugins we support.
+
+// Additional middleware packages
+//go:generate go run ../gen/directives_generate.go ../middleware.cfg
```

Comment:
directives_generate.go will generate the imports or directives based on the package go:generate was called from